api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -343,7 +343,7 @@ public FirecloudWorkspace cloneWorkspace(
                 ImmutableList.of(new FirecloudManagedGroupRef().membersGroupName(authDomainName)));
 
     return retryHandler.run(
-        (context) -> workspacesApi.cloneWorkspace(fromProject, fromName, cloneRequest));
+        (context) -> workspacesApi.cloneWorkspace(cloneRequest, fromProject, fromName));
```

Comment:
Can you clarify why this changed? Is the order different in the firecloud yaml?

If you haven't already, please check any usages of these APIs to verify there aren't any ordering changes which weren't caught by compilation - for example foo(String x, String y) -> foo(String y, String x)